### ISSUE TITLE
Training on custom Dataset

### ISSUE THREAD

**Issue by saifsayed** [ShuweiShao/AF-SfMLearner#5]
Dear Author, 
Congratulations on the publication and thank you for sharing the code. This is a great contribution to computer vision.

I am a PhD student and was trying to use the model for training on a new dataset. Can you please guide me on aspects I need to take care of while training.

1. Do I need to train from scratch or finetune the existing trained model.
2. I have videos with different intrinsics, so do I have to update the intrinsic parameter in the training loop or train model stage-wise keeping intrinsics constant.
3. Which training strategy you would prefer? Stage-wise or end-to-end

**Comment by ShuweiShao** [comment]
Sorry for the late reply!
1. you could try both manners, and compare the accuracy.
2. I think both may be OK.
3. The end-to-end is more convenient.
